### ISSUE TITLE
src/hot-table.html should be moved to ./hot-table.html

### ISSUE THREAD

**Issue by MetaMemoryT** [handsontable/hot-table#18]
All polymer elements in https://github.com/PolymerElements have the main importable element (in this case hot-table.html) at the top-level directory of the project.

This convention makes importing predictable, you just include:

```
<link rel="import" href="bower_components/paper-button/paper-button.html">
```

Rather than each element having something different and un-predictable like:

```
<link rel="import" href="bower_components/paper-button/src/paper-button.html">
<link rel="import" href="bower_components/paper-button/dist/paper-button.html">
```

etc.

**Comment by devmikko** [comment]
:+1:

**Comment by budnix** [comment]
Released in v0.2.0
